packages/fiori-elements-writer/test/__snapshots__/ovp.test.ts.snap

```diff
@@ -8843,7 +8843,7 @@ appDescription=A Fiori application.",
   },
   \\"sap.ovp\\": {
     \\"globalFilterModel\\": \\"mainModel\\",
-    \\"globalFilterEntityType\\": \\"SalesOrderItem\\",
+    \\"globalFilterEntitySet\\": \\"SalesOrderItem\\",
```

Comment:
I would expect the value to also have changed since this is the entity type rather than entity set?